beacon-chain/sync/rate_limiter.go

```diff
@@ -59,6 +59,9 @@ func newRateLimiter(p2pProvider p2p.P2P) *limiter {
 	// Collector for V2
 	blockCollectorV2 := leakybucket.NewCollector(allowedBlocksPerSecond, allowedBlocksBurst, blockBucketPeriod, false /* deleteEmptyBuckets */)
 
+	// for BlobSidecarsByRoot and BlobSidecarsByRange
+	blobCollector := leakybucket.NewCollector(allowedBlocksPerSecond, allowedBlocksBurst, blockBucketPeriod, false)
```

Comment:
is it ok to assume 1-1 mapping for blob rate limiter params( same as for a block).

Reply:
Good point, it's probably better for this to have its own set of params that can be tuned independently.